lib/asciidoctor/cli/options.rb

```diff
@@ -137,6 +138,9 @@ def parse!(args)
           opts.on('-t', '--timings', 'print timings report (default: false)') do |timing|
             self[:timings] = true
           end
+          opts.on('--sourcemap', 'generate source_location data in the AST. Usefull for extensions (default: false)') do |sourcemap|
```

Comment:
I think it's "Useful" with one "l":
```suggestion
          opts.on('--sourcemap', 'generate source_location data in the AST. Useful for extensions (default: false)') do |sourcemap|
```

Reply:
The description might be too low level. For reference, here's the description from the User Manual:

> Keeps track of the file and line number for each parsed block. (Useful for tooling applications where the association between the converted output and the source file is important).

I think we should first describe what it does and then where we can retrieve the data?

Reply:
I aggree. I changed the description --help returns to `keep track of the file and line number for each parsed block (default:false)` and made the full description in the asciidoctor documentation the same as in the User Manual.